src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,943 @@
+  </div>
+
+## Locking reads
+
+To reduce the occurrence of [concurrency anomalies](#concurrency-anomalies) in `READ COMMITTED` isolation, you can strengthen the isolation of individual transactions by using [`SELECT FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) or [`SELECT FOR SHARE`]({% link {{ page.version.version }}/select-for-update.md %}) to issue *locking reads* on specific rows. Locking reads behave similarly to [writes]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#write-intents): they lock qualifying rows to prevent concurrent writes from modifying them until the transaction commits. Conversely, if a locking read finds that a row is exclusively locked by a concurrent transaction, it waits for the other transaction to commit or rollback before proceeding. A locking read in a transaction will always have the latest value of a row when the transaction commits.
+
+The clause used with the `SELECT` statement determines the *lock strength* of a locking read:
+
+{% include {{ page.version.version }}/sql/select-lock-strengths.md %}
+
+By default for `READ COMMITTED` transactions, locks are replicated via [Raft]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft). This allows locks to be preserved even if the [leaseholder node]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leases) serving a locking read dies and its [lease is transferred to another node]({% link {{ page.version.version }}/architecture/replication-layer.md %}#how-leases-are-transferred-from-a-dead-node).
+
+### When to use locking reads
+
+Use locking reads in your application if certain `READ COMMITTED` transactions must guarantee data correctness in [high-contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention) scenarios.
+
+Non-locking reads can allow intermediate writes to update rows before `READ COMMITTED` transactions commit, potentially creating [concurrency anomalies](#concurrency-anomalies). Locking reads prevent such anomalies, but increase the amount of lock contention that [may require intervention]({% link {{ page.version.version }}/performance-recipes.md %}#waiting-transaction) if latency becomes too high.
+
+Locking reads are not effective for emulating `SERIALIZABLE` transactions, which can avoid locking reads because they [refresh reads]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#read-refreshing) at commit time to verify their correctness, potentially causing the transaction to abort. As a result, `READ COMMITTED` transactions that use locking reads will perform differently than `SERIALIZABLE` transactions at various levels of concurrency.
+
+If you decide to use locking reads:
+
+- If a transaction needs to update a row, use `SELECT FOR UPDATE` to acquire an exclusive lock on the row. This guarantees data integrity between the transaction's read and write operations.
```

Comment:
"Needs to read a row and then later update it in the same transaction"

We wouldn't want users thinking that they need to SELECT FOR UPDATE first before issuing any UPDATE statement.